pom.xml

```diff
@@ -564,7 +564,7 @@
             This line enables code-coverage. However, it also blows the build's permgen.
             Fix me!
           -->
-          <argLine>-XX:-UseSplitVerifier ${jacoco.agent.unit.arg}</argLine>
+          <argLine>-XX:-UseSplitVerifier -XX:MaxMetaspaceSize=512m ${jacoco.agent.unit.arg}</argLine>
```

Comment:
We can probably also remove `UseSplitVerifier`
http://stackoverflow.com/questions/31566672/java-8-what-is-the-equivalent-of-usesplitverifier